packages/rum-core/src/boot/rum.ts

```diff
@@ -90,6 +100,8 @@ export function startRumEventCollection(
       // prevent batch from previous tests to keep running and send unwanted requests
       // could be replaced by stopping all the component when they will all have a stop method
       batch.stop()
+      parentContexts.stop()
```

Comment:
Was this missing from the current SDK?

Reply:
it is only used for test, nice catch though 😉